portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase

```diff
@@ -0,0 +1,54 @@
+@component-name = "portal-wcm"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property test.prepare.bundle.properties = "false";
+	property testray.main.component.name = "Content Pages";
+
+	@description = "This is a test for LPS-134072. View the Home page when start up with Spanish as the default instance language."
+	@priority = "4"
+	test ViewHomePageWithSpanishAsDefaultLanguage {
+		property database.types = "hypersonic";
+		property test.name.skip.portal.instance = "ContentePagesWithSetupWizard#ViewHomePageWithSpanishAsDefaultLanguage";
+
+		task ("Configure the user") {
+			SetupWizard.configureAdministratorUserPG(
+				administratorEmail = "user@liferay.com",
+				administratorFirstName = "User",
+				administratorLastName = "User");
+		}
+
+		task ("Select Spanish as default language") {
+			SetupWizard.configureDefaultLanguagePG(defaultLanguage = "español (España)");
+
+			SetupWizard.finishConfigurationPG();
+		}
+
+		task ("Accept the user license agreement") {
+			User.acceptEndUserLicenseAgreementLocalized(localizedAgree = "Acepto");
+		}
+
+		task ("Configue the password and the reminder") {
+			User.resetPassword();
+
+			User.answerPasswordReminderLocalized(
+				localizedAnswer = "Respuesta",
+				localizedSaveButton = "Guardar");
+		}
+
+		task ("View default welcome content page") {
+			AssertTextEquals(
+				key_fragmentName = "heading",
+				locator1 = "Fragment#FRAGMENT_BASIC_COMPONENT",
+				value1 = "Bienvenido a Liferay");
+
+			AssertVisible(
+				id = "image-square",
+				key_image = "tree.png",
+				key_imageDescription = "Responsive Image",
+				locator1 = "Fragment#CONTRIBUTED_FRAGMENT_EDITABLE_FIELD_IMAGE_PROPERTIES");
+		}
```

Comment:
Only keep these two steps, then move to Locale.RestrictDefaultLocale

Reply:
> Only keep these two steps, then move to Locale.RestrictDefaultLocale

Hi @Tim-Cao, I tried to put these two steps on the file above like you said but it's not working. Could you show me how it supposed to be? Thanks in advance.

I think it's just going to work if I set the basic configuration to Spanish manually and then run the Locale.RestrictDefaultLocale, but the description says to automate all process.

Reply:
Did you add company.default.locale=es_ES to your portal-ext.properties ?

Reply:
I didn't, it's the first time that I'm doing a test with the basic configuration. I'll add and try to do it again.